src/System.Threading.Tasks/tests/System.Runtime.CompilerServices/AsyncTaskMethodBuilderTests.cs

```diff
@@ -345,6 +345,7 @@ public static void TaskMethodBuilderT_TaskIsCached()
         }
 
         [Fact]
+        [ActiveIssue("TFS 450361 - Codegen optimization issue", TargetFrameworkMonikers.UapAot)]
         public static void TaskMethodBuilder_UsesCompletedCache()
```

Comment:
I reused the branch from the previous PR and GitHub shows the previous commit.  Changes in this file have been already merged.